### PR TITLE
Display additional information for visible Pokemons in console

### PR DESCRIPTION
**Pull request by NgJason**
This pull request includes a:
- [x] New feature/ Enhancement

Display Pokemon name, coordinate, ID and remaining time for any discovered Pokemons. This is useful for users who do not need to look at the map. With the output, users can copy and paste the coordinate into any external program/gps.

Usage: 
Add the flag -dc

Ex Output in console:
<p>
===============================<br>
 Name: Zubat<br>
 Coord: (37.777523,-122.400932)<br>
 ID: 41<br>
 Remaining Time: 00m23s<br>
===============================
</p>

### CODE REVIEW

**Comment by POPSuL** [comment]
Might be worth to break the 708 line into two lines?

File: example.py [hunk 1]
```diff
@@ -692,9 +698,15 @@ def process_step(args, api_endpoint, access_token, profile_response,
...
 
-        disappear_timestamp = time.time() + poke.TimeTillHiddenMs \
+        curTime = time.time()
+        disappear_timestamp = curTime + poke.TimeTillHiddenMs \
             / 1000
 
+        if args.display_in_console:
+            ls = disappear_timestamp-curTime
+            m, s = divmod(ls, 60)
+            print "======================================\n Name: %s\n Coord: (%f,%f)\n ID: %s \n Reamining Time: %02dm%02ds\n======================================" % (pokename.encode('utf-8'),poke.Latitude,poke.Longitude,pokeid,m,s)
```
Comment (myran2) [inline example.py]: Reamining Time: -> Remaining Time:

**Comment by myran2** [comment]
You should fix the typo on line 708 too. 'Reamining' should be 'Remaining'.

**Comment by NgJason** [comment]
Resolved conflict with base branch. Add the following to the wiki Usage page,
| `-dc` `--display-in-console`| Display found Pokémon name, coordinates, ID, and remaining time |          |

**Pull request closed without merging**